.github/workflows/full-qa.yml

```diff
@@ -0,0 +1,135 @@
+name: Full QA
+
+on:
+  # new commits pushed to release branches
+  push:
+    branches:
+      - "master" # pushes to the `master` branch
+      - "*.*" # pushes to the released version branches (e.g. `1.0`)
+
+jobs:
+  unit-tests-p2:
+    name: Unit Tests (Polymer 2)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node 12.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+
+      - name: Check out the source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Install global npm dependencies
+        # bower is needed to run 'bower install'
+        # polymer-cli is needed to run the lint step
+        # web-component-tester is needed to run the test step
+        run: "npm install --quiet --no-progress --global bower polymer-cli web-component-tester"
+
+      - name: Install project npm dependencies
+        run: "npm install --quiet --no-progress"
+
+      - name: Install project Bower dependencies
+        run: "bower install --quiet"
+
+      - name: Run automated magi-cli checks
+        run: "npm run check"
+
+      - name: Run a linter
+        run: "npm run lint"
+
+      - name: Run unit tests on SauceLabs
+        if: ${{ github.event != 'schedule' }}
+        run: "wct --env saucelabs"
+        env:
+          SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+
+      - name: Run unit tests on SauceLabs (cron)
+        if: ${{ github.event == 'schedule' }}
+        run: "wct --env saucelabs-cron"
+        env:
+          SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
+
+  unit-tests-p3:
+    name: Unit Tests (Polymer 3)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Node 12.x
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
+
+      - name: Check out the (Polymer 2) source code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Same as above, this part can be removed.

Reply:
removed